drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -507,7 +507,7 @@ static void usbd_enable_endpoints(struct nrf_usbd_ctx *ctx)
 	struct nrf_usbd_ep_ctx *ep_ctx;
 	int i;
 
-	for (i = 0; i < NRF_USBD_EPIN_CNT; i++) {
+	for (i = 0; i < CFG_EPIN_CNT; i++) {
```

Comment:
I hope I do not overlook anything, but I think you do not need DT to describe number of the endpoints for this design or is there (or will be) a design with the same IP but different number of endpoints?